public-api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -100,6 +108,109 @@ do
+  gvs_amr_ac, gvs_amr_an, gvs_amr_af,
+  gvs_eas_ac, gvs_eas_an, gvs_eas_af,
+  gvs_mid_ac, gvs_mid_an, gvs_mid_af,
+  gvs_eur_ac, gvs_eur_an, gvs_eur_af,
+  gvs_sas_ac, gvs_sas_an, gvs_sas_af,
+  gvs_oth_ac, gvs_oth_an, gvs_oth_af,
+  ROW_NUMBER() OVER(PARTITION BY vid ORDER BY
+     CASE ARRAY_TO_STRING(consequence, ',')
+         WHEN 'upstream_gene_variant'
+         THEN 4
+         WHEN 'downstream_gene_variant'
+         THEN 5
+         ELSE 1
+         END
+         ASC
+  )  AS row_number
+  FROM \`$OUTPUT_PROJECT.$GENOMICS_DATASET.vat_v2\`
+  WHERE is_canonical_transcript OR transcript is NULL
+  ORDER BY vid, row_number),
+  genes as (
+     SELECT vid, ARRAY_TO_STRING(array_agg(distinct gene_symbol ignore nulls ORDER BY gene_symbol), ' ') as genes
+     FROM \`$OUTPUT_PROJECT.$GENOMICS_DATASET.vat_v2\`
+     GROUP BY vid
+  )
+  SELECT
+     sorted_transcripts.variant_id,
+     sorted_transcripts.gene_symbol,
+     sorted_transcripts.consequence,
+     sorted_transcripts.protein_change,
+     sorted_transcripts.dna_change_in_transcript as dna_change,
+     sorted_transcripts.gvs_all_ac as allele_count,
+     sorted_transcripts.gvs_all_an as allele_number,
+     sorted_transcripts.gvs_all_af as allele_frequency,
+     sorted_transcripts.clinical_significance,
+     sorted_transcripts.rs_number,
+     sorted_transcripts.transcript as transcript,
+     sorted_transcripts.gvs_afr_ac,
+     sorted_transcripts.gvs_afr_an,
+     sorted_transcripts.gvs_afr_af,
+     sorted_transcripts.gvs_amr_ac,
+     sorted_transcripts.gvs_amr_an,
+     sorted_transcripts.gvs_amr_af,
+     sorted_transcripts.gvs_eas_ac,
+     sorted_transcripts.gvs_eas_an,
+     sorted_transcripts.gvs_eas_af,
+     sorted_transcripts.gvs_mid_ac,
+     sorted_transcripts.gvs_mid_an,
+     sorted_transcripts.gvs_mid_af,
+     sorted_transcripts.gvs_eur_ac,
+     sorted_transcripts.gvs_eur_an,
+     sorted_transcripts.gvs_eur_af,
+     sorted_transcripts.gvs_sas_ac,
+     sorted_transcripts.gvs_sas_an,
+     sorted_transcripts.gvs_sas_af,
+     sorted_transcripts.gvs_oth_ac,
+     sorted_transcripts.gvs_oth_an,
+     sorted_transcripts.gvs_oth_af,
+     sorted_transcripts.contig,
+     sorted_transcripts.position,
+     sorted_transcripts.ref_allele,
+     sorted_transcripts.alt_allele,
+     sorted_transcripts.cons_str,
+     genes.genes
+  FROM sorted_transcripts, genes
+  WHERE genes.vid = sorted_transcripts.variant_id
+  AND (sorted_transcripts.row_number =1 or sorted_transcripts.transcript is NULL);"
+
+  bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+  "DROP TABLE IF EXISTS \`$OUTPUT_PROJECT.$GENOMICS_DATASET.wgs_variant_vid_cluster\`"
+
+  bq mk --table --schema=$schema_path/wgs_variant_vid_cluster.json --time_partitioning_type DAY --clustering_fields variant_id aou-db-prod:genomics_v1.wgs_variant_vid_cluster
+
+  bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+  "insert into \`$OUTPUT_PROJECT.$GENOMICS_DATASET.wgs_variant_vid_cluster\`
```

Comment:
This should be a reclustered materialized view off the base table wgs_variant. You do this in the next step. You only need the MV but not this concrete table.

Reply:
Changed this.